src/main/kotlin/anttracker/Screen.kt

```diff
@@ -1,76 +1,5 @@
 package anttracker
```

Comment:
Me, Tyrus, and Angela will be using functions to represent screens; will this be used just in Issue?

Reply:
That was my original plan.